src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    Circle(String color, int radius) {
```

Comment:
Constructors should be public to allow object creation from outside the package.